src/main/java/de/wi2020sebgroup1/cinema/entities/Seat.java

```diff
@@ -42,7 +42,7 @@
 	
 	@Column
 	@NotNull
-	private int priceMultiplier;
+	private double priceMultiplier;
```

Comment:
Ich weiß nicht ob das funktioniert. 
muss ausprobieret werden. Ist mir gegen die Wand gefahren

Reply:
Weil nen multiplier wäre an der Stelle hier ja wahrscheinlich nur sowas wie 1,1 oder 1,5 oder so und nicht direkt 2 oder 3

Reply:
Wenn das nicht geht ist der Multiplier einfach eine Ganzzahl die den PPreisaufschlag in Prozent anzeigt (zb 20 für 1,2 bzw für 120%)

Reply:
So hatte ich mir das gedacht also 20 für 1,2 oder 200, wenn es doch gleich der 2x Preis ist. Ich weiß nicht wie gut die DB mit Double-Werten umgehen kann.

Reply:
Und einfache ints brauchen glaube ich weniger Speicherplatz... muss ich mal prüfen, oder du. 
An sich ist mir ein Double auch recht, wenn es funktioniert